packages/components/molecules/f-restaurant-card/src/components/subcomponents/_tests/IconText.test.js

```diff
@@ -0,0 +1,88 @@
+import { shallowMount } from '@vue/test-utils';
+import sut from '../IconText.vue';
+
+describe('IconText.vue', () => {
+    it('displays the provided text', () => {
+        // arrange
+        const propsData = {
+            text: 'foo'
+        };
+
+        // act
+        const wrapper = shallowMount(sut, { propsData });
+
+        const visibleText = wrapper.find('[data-test-id="icon-text-visible-text"]').text();
+
+        // assert
+        expect(visibleText).toBe('foo');
+    });
+
+    it('displays the provided accessible Text', () => {
+        // arrange
+        const propsData = {
+            text: 'foo',
+            accessibleText: 'bar'
+        };
+
+        // act
+        const wrapper = shallowMount(sut, { propsData });
+
+        const accessibleText = wrapper.find('[data-test-id="icon-text-accessible-text"]').text();
+
+        // assert
+        expect(accessibleText).toBe('bar');
+    });
+
+    it('fallbacks to text if accessible text is missing', () => {
+        // arrange
+        const propsData = {
+            text: 'foo'
+        };
+
+        // act
+        const wrapper = shallowMount(sut, { propsData });
+
+        const accessibleText = wrapper.find('[data-test-id="icon-text-accessible-text"]').text();
+
+        // assert
+        expect(accessibleText).toBe('foo');
+    });
+
+    it('when icon is present and hideInTileView is true and in tile mode, hides Icon', () => {
```

Comment:
This is quite a complicated test case (in it's wording). Might be worth creating a describe block for the icon rendering, then a case when tile mode and hide are true etc

Reply:
Fixed 😄